src/interface/src/app/plan/plan.component.ts

```diff
@@ -137,10 +137,8 @@ export class PlanComponent implements OnInit, OnDestroy {
     const path = routeChild?.url[0].path;
     const id = routeChild?.paramMap.get('id') ?? null;
 
-    if (path === 'scenario') {
+    if (path === 'config') {
       this.planService.updateStateWithScenario(id);
-    } else if (path === 'config') {
-      this.planService.updateStateWithConfig(Number(id));
```

Comment:
Are we still using the method `planService.updateStateWithConfig` somewhere else? Should we not use it anymore? Im always a bit confused about scenarios vs configs.
Can we remove that method from `planService`? or if its a bigger refactor effort, mark it as deprecated?

Reply:
yeah Config (as in the currentConfig in planstate) can be deprecated. I added some comments and a todo to remove